index.js

```diff
@@ -1,43 +1,48 @@
 const BrowserUtils = require('./lib/BrowserUtils');
 const NativeUtils = require('./lib/NativeUtils');
-const wdioUI5 = require("./lib/wdioUi5-index");
-const logger = require("./lib/Logger");
-const selectorHelper = require("./lib/SelectorHelper");
+const wdioUI5 = require('./lib/wdioUi5-index');
+const cordovaPluginFactory = require('./lib/cordova-plugins/factory');
```

Comment:
for better context + understanding, probably rename to "cordovaMockPluginFactory" or similar?

Reply:
- renamed folder "cordova-plugins" to "cordova-plugin-mocks"
- changed variable name